rnote-ui/src/appwindow/appwindowactions.rs

```diff
@@ -302,18 +302,6 @@ impl RnAppWindow {
             }),
         );
 
-        // Restrict Zoom
-        action_block_pinch_zoom.connect_change_state(
-            clone!(@weak self as appwindow => move |action_block_pinch_zoom, state_request| {
-                let block_pinch_zoom = state_request.unwrap().get::<bool>().unwrap();
-                let active_tab = appwindow.active_tab();
-
-                active_tab.set_block_pinch_zoom(block_pinch_zoom);
-
-                action_block_pinch_zoom.set_state(block_pinch_zoom.to_variant());
-            }),
-        );
-
```

Comment:
I removed it since I am not sure what it does and if its nessary for PropertyActions. I found no similar code for touch-drawing

Reply:
It seems to work without it.

Reply:
If it's not overwritten by a custom callback, the default changes the action state (so: the property) in the same way